recipes/rmlui/3.3/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class RmluiConan(ConanFile):
+    name = "rmlui"
+    description = "RmlUi - The HTML/CSS User Interface Library Evolved"
+    homepage = "https://github.com/mikke89/RmlUi"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    topics = ("conan", "css", "gui", "html", "lua", "rmlui")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "build_lua_bindings": [True, False],
+        "disable_rtti_and_exceptions": [True, False],
+        "enable_precompiled_headers": [True, False],
+        "enable_tracy_profiling": [True, False],
+        "fPIC": [True, False],
+        "no_font_interface_default": [True, False],
```

Comment:
- Please don't use double negation
- What is the default when no interface is defaulted? Can the default be set?

Reply:
My approach with all of the available options was to name them as they are in the library's documentation. I do agree that it doesn't say much without reading the documentation. In order to actually convey something here, I'd rename it to "with_freetype_font_interface", since if this is turned off then the user is free to implement their own. Would this be acceptable to you?

Reply:
How about the following?
```python
"font_interface": ["freetype", None],
```
it allows for future expansion without adding a zillion options that can cause conflicts.

Reply:
Sounds great!